src/test/java/me/itzg/tsdbcassandra/services/MetadataServiceTest.java

```diff
@@ -154,64 +143,6 @@ void getTenants() {
     assertThat(result).containsExactlyInAnyOrderElementsOf(tenantIds);
   }
 
-  @Nested
-  class metricNameHasAggregator {
-
-    @Test
-    void exists() {
-      final String tenant = RandomStringUtils.randomAlphanumeric(10);
-      final String metricName = RandomStringUtils.randomAlphanumeric(10);
-
-      insertOne(tenant, metricName);
-
-      final Boolean result = metadataService
-          .metricNameHasAggregator(tenant, metricName, Aggregator.min)
-          .block();
-
-      assertThat(result).isTrue();
-    }
-
-    private void insertOne(String tenant, String metricName) {
-      cassandraTemplate.insert(
-          new MetricName()
-              .setTenant(tenant)
-              .setMetricName(metricName)
-              .setAggregators(Set.of(Aggregator.raw, Aggregator.min, Aggregator.max))
-      ).block();
-    }
-
-    @Test
-    void existsButNotAggregator() {
-      final String tenant = RandomStringUtils.randomAlphanumeric(10);
-      final String metricName = RandomStringUtils.randomAlphanumeric(10);
-
-      insertOne(tenant, metricName);
-
-      final Boolean result = metadataService
-          .metricNameHasAggregator(tenant, metricName, Aggregator.avg)
-          .block();
-
-      assertThat(result).isFalse();
-    }
-
-    @Test
-    void notExists() {
-      final String tenant = RandomStringUtils.randomAlphanumeric(10);
-      final String metricName = RandomStringUtils.randomAlphanumeric(10);
-
-      insertOne(tenant, metricName);
-
-      final Boolean result = metadataService
-          .metricNameHasAggregator(tenant,
-              // query for metric name that doesn't exist
-              metricName+"_not_exists",
-              Aggregator.min)
-          .block();
-
-      assertThat(result).isFalse();
-    }
-  }
-
```

Comment:
these tests don't seem to exist.  do you want to add some thing like this?:
```
  @Test
  void locateSeriesSets() {
    Assertions.fail("TODO");
  }
  @Test
  void getMetricNames() {
    Assertions.fail("TODO");
  }
  @Test
  void getTagKeys() {
    Assertions.fail("TODO");
  }
  @Test
  void getTagValues() {
    Assertions.fail("TODO");
  }
```

Reply:
Ah, yes, especially since that's what I did in other spots to remind me to add tests later.